release-manager/src/main/java/com/escuela/releasemanager/deserializer/JiraIssueDeserializer.java

```diff
@@ -0,0 +1,43 @@
+package com.escuela.releasemanager.deserializer;
+
+import com.escuela.releasemanager.model.IssueModel;
+import com.escuela.releasemanager.model.IssueSearchModel;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class JiraIssueDeserializer extends JsonDeserializer<IssueSearchModel> {
+
+    public JiraIssueDeserializer() {
+    }
+
+    @Override
+    public IssueSearchModel deserialize(JsonParser jsonParser, DeserializationContext deserializationContext) throws IOException {
+        List<IssueModel> issueList = new ArrayList<>();
+        IssueSearchModel issueSearchModel = new IssueSearchModel();
+        JsonNode jiraIssueResponseNode = jsonParser.getCodec().readTree(jsonParser);
```

Comment:
How you handling the exception while parsing ?

Reply:
It should based on your application exception handling strategy.Since spring provide default exception handler , application won't down like in traditional pure java applications.
But I could create Custom Exception and throw with a message like "Something went Wrong With Jira Server".Is that OK?

Reply:
Any other idea?

Reply:
Add global exception handler with custom exception.